docs/getting-started.md

```diff
@@ -8,6 +8,12 @@
 yarn add @dhis2/ui
 ```
 
+In some cases you might need to install `styled-jsx` as peer dependency. To do so run
+
+```bash
+yarn add styled-jsx -P
+```
```

Comment:
With regards to the comment above, we could add all three peer dependencies

Reply:
yep